CHANGELOG.md

```diff
@@ -49,6 +49,7 @@
 * [ENHANCEMENT] Querier: reduce memory consumed for queries that hit store-gateways. #6348
 * [ENHANCEMENT] Ruler: include corresponding trace ID with log messages associated with rule evaluation. #6379
 * [ENHANCEMENT] Querier: clarify log messages and span events emitted while querying ingesters, and include both ingester name and address when relevant. #6381
+* [ENHANCEMENT] Ingester, Distributor: added experimental support for rejecting push requests received via gRPC before reading them into memory, if ingester or distributor is unable to accept the request. This is activated by using `-ingester.limit-inflight-requests-using-grpc-method-limiter` for ingester, and `-distributor.limit-inflight-requests-using-grpc-method-limiter` for distributor. #5976 #6300
```

Comment:
Mention the experimental flags in `docs/sources/mimir/configure/about-versioning.md` too, please.

Reply:
https://github.com/grafana/mimir/pull/6300/commits/304de04c7169aed6b1e4330c07e7f06077c853c7